apps/coscrad-frontend/src/store/slices/resources/shared/notes-for-resource/self-note.presenter.tsx

```diff
@@ -1,15 +1,28 @@
+import styled from '@emotion/styled';
+import { TextSnippet as TextSnippetIcon } from '@mui/icons-material';
+import { Box, Grid, Typography } from '@mui/material';
 import { SelfConnectionNote } from '../../../notes/hooks/use-loadable-self-notes-for-resource';
 
+const Item = styled(Box)(({ theme }) => ({
+    padding: 0,
+    marginBottom: '15px',
+}));
+
 export const SelfNotePresenter = ({ text, id, context }: SelfConnectionNote): JSX.Element => (
-    <div data-testid={id}>
-        {/* TODO Use property presenter helper after rebasing */}
-        <strong>Note ({id}):</strong>
-        <p>{text}</p>
-        <br />
-        {/* TODO Eventually we should offer a way to highlight this context in the detail view */}
-        {/* We may not need to present this here in this way. This is for development only. */}
-        <strong>Context</strong>
-        {JSON.stringify(context)}
-        <br />
-    </div>
+    <Item data-testid={id}>
+        <Grid container spacing={0} columns={2}>
+            <Grid item sx={{ pr: 2 }}>
+                <TextSnippetIcon />
+            </Grid>
+            <Grid item>
+                {/* TODO Use property presenter helper after rebasing */}
+                <Typography variant="h5">Note ({id}):</Typography>
```

Comment:
need a presenter like `SinglePropertyPresenter` for properties that have a paragraph or value underneath the label which is itself a header.  Different than our existing presenter where the label is on the left and value on the right.

Reply:
Sounds good. You might wait until you need this again.